src/celeritas/em/interactor/CoulombScatteringInteractor.hh

```diff
@@ -92,21 +98,31 @@ class CoulombScatteringInteractor
  */
 CELER_FUNCTION
 CoulombScatteringInteractor::CoulombScatteringInteractor(
-    CoulombScatteringRef const& shared,
+    CoulombScatteringData const& shared,
+    NativeCRef<WentzelOKVIData> const& wentzel,
     ParticleTrackView const& particle,
     Real3 const& inc_direction,
+    MaterialView const& material,
     IsotopeView const& target,
-    ElementId const& el_id,
+    ElementId el_id,
     CutoffView const& cutoffs)
     : inc_direction_(inc_direction)
     , particle_(particle)
     , target_(target)
-    , sample_angle(particle,
-                   target,
-                   shared.elem_data[el_id],
-                   // TODO: Use proton when supported
-                   cutoffs.energy(shared.ids.electron),
-                   shared)
+    , helper_(particle,
+              material,
+              target.atomic_number(),
+              wentzel.params,
+              shared.ids,
+              // TODO: Use proton when supported
```

Comment:
I think this todo is for changing the _cutoff_ to be for protons instead of electrons: https://github.com/celeritas-project/celeritas/pull/861#discussion_r1288852391

Reply:
Okay - protons will  not be supported in this model, but the recoil nucleus (with the minimal proton production cut) should be included properly when when (?? maybe never) we support generic ions in the future.  So, the better comment may be "Use the proton production cutoff when the recoiled nucleus production is supported".

Reply:
Thanks, updated the comment.